### PR TITLE
InstantSearch v1

### PR DESCRIPTION
**Pull request by mattfredericksen**

### CODE REVIEW

File: apps/entities/api/views.py [hunk 1]
```diff
@@ -19,3 +23,12 @@ class EntityListAPIView(ListAPIView):
     permission_classes = [permissions.AllowAny]
     serializer_class = EntitySerializer
     queryset = Entity.objects.exclude(type='entities.deity').order_by('birth_date')  # type: ignore
+
+class EntityInstantSearchAPIView(APIView):
+
```
Comment (iacobfred) [inline apps/entities/api/views.py:28]: docstring?

File: frontend/components/search/InstantSearch.tsx [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+  const handleValueChange = (event, values) => {
+    setState((prevState) => ({ ...prevState, [name]: values.map((value) => value[idKey]) }));
+    setSelectedOptions(values);
+  };
+
+  // Throttling behavior only works if the same instance of
+  // the throttled callback is used, so we use `useCallback`.
+  // eslint-disable-next-line
+  const getThrottledDataForInput = useCallback(
+    throttle(throttleDelay, (...args: Parameters<typeof getDataForInput>) =>
```
Comment (iacobfred) [inline frontend/components/search/InstantSearch.tsx:79]: Cool!  What is the throttling behavior?

**Review by iacobfred — APPROVED** [review]
stupendous

File: frontend/components/search/InstantSearch.tsx (hunk 2 above)
Comment (mattfredericksen) [reply frontend/components/search/InstantSearch.tsx:92]: It forces a delay between API requests for efficiency reasons. Not super important right now, but the structure is in place for when application usage increases.

**Review by iacobfred — APPROVED** [review]
凄まじい

**Pull request merged**